### ISSUE TITLE
Review accessibility guidance on page titles 

### ISSUE THREAD

**Issue by lucyha** [nhsuk/nhsuk-service-manual#1018]
## What

As part of the review of our accessibility pages, the guidance for content on setting page titles ([linked here](https://service-manual.nhs.uk/accessibility/content)) needs to be reviewed.

## Why

The guidance needs to clarify how page titles differ from page names.

## Done when

- [ ] New guidance drafted
- [ ] Guidance reviewed and approved
- [ ] Updated guidance released

**Comment by sarawilcox** [comment]
Hi @AJSMITH1512, here's the current guidance: https://service-manual.nhs.uk/accessibility/content#set-page-titles

Is the following text clearer? I'll run it by a few content designers this week, and will check it with Dave and Ben too before asking the AWG to approve the change.

**Set page names and page titles**
For: Content, Development, Testing

The page name:
- is in the <H1> tag in the <body> of the page, for example: "Achalasia"
- shows at the top of the page

The page title:
- is in the <title> tag in the <head> section of the page
- consists of the page name and the website name, like this: Page name – Site name, for example: "Achalasia – NHS"
- shows in the browser window and search results snippet
- is the first thing a screen read will read out

If you can, use templates to keep things consistent and use your content management system (CMS) to add the site name after the dash. That way you can focus on the page name in the H1.

Good page names:
- help users find what they want and recognise they're in the right place
- are unique, concise and descriptive
- put important keywords near the beginning

**Comment by sarawilcox** [comment]
Should probably go to AWG too. I'm testing it with some content designers this morning.

**Comment by sarawilcox** [comment]
### Notes from crit with content designers
- Confusion about words "page name" and "page title". Explain up front that they can be confused and then explain key differences? Better to talk about "title" and "H1" or main heading.
- Consider putting content for content designers under "Use headings correctly" and leave "Set page titles" for devs and testers? But some content designer duplicate H1 in title or write different page titles. Some are just aware of and use H1 and rely on CMS creating page title.
- For content designers, it's about "writing good page titles". "Set" doesn't make sense to them. In what sense to devs "set" page titles?
- When would we have different info in page title and H1? (See Slack conversation.)
- Make it clear that title doesn't appear on page but does appear in search results. While H1 is visual, on page. 
- Consider using accordions for what devs need versus what content designers need?
- Section heading: Set page titles and write good page headings?

**Comment by sarawilcox** [comment]
See Slack thread: https://nhsuk.slack.com/archives/C6S0QTW9X/p1624370799028200?thread_ts=1624368546.027100&cid=C6S0QTW9X

**Comment by sarawilcox** [comment]
We say this in the accessibility guidance: https://service-manual.nhs.uk/accessibility/content#set-page-titles, i.e. Page title - Site name. For example: Achalasia - NHS.
I think we're OK on the health content front, because it's templated in Wagtail. But we're not consistent with services.
We have some services that follow the GOV.UK pattern: eg: Do you know your NHS number? - Book a coronavirus vaccination - NHS. https://www.nhs.uk/book-a-coronavirus-vaccination/do-you-have-an-nhs-number.
And some which don't mention that it's the NHS website, e.g.: Search - Find a walk-in coronavirus (COVID-19) vaccination site. https://www.nhs.uk/service-search/find-a-walk-in-coronavirus-covid-19-vaccination-site

**Comment by sarawilcox** [comment]
Had a chat with Alistair Duggin and Sarah Norman about this. We know that some transactional services are failing WCAG2.1 AA on page titles. Other pages are not following latest best practice.

Alistair is going to draft a brief note with examples and best practice that could go in the accessibility guidance.

We also need a way to [make this guidance easier to find in the service manual](https://github.com/nhsuk/nhsuk-service-manual/issues/1356). It's not just an accessibility issue.

**Comment by sarawilcox** [comment]
This all needs further review and testing. Put back into team backlog for now.

**Comment by sarawilcox** [comment]
See also our draft URL guidance: https://github.com/nhsuk/nhsuk-service-manual-community-backlog/issues/265

The BBC guidance for title is
Page name - Section name - BBC

And the [GOV.UK](http://gov.uk/) guidance (if I remember correctly) is:
Page name - service name - [GOV.UK](http://gov.uk/)
eg https://www.registertovote.service.gov.uk/register-to-vote/country-of-residence
![image (5)](https://github.com/nhsuk/nhsuk-service-manual/assets/45792959/b3e866ef-df9d-4d41-988d-234c06751ee0)

**Comment by sarawilcox** [comment]
Page titles should be short. WebAim says generally no more than a few words. https://webaim.org/techniques/pagetitle/

DigitalA11y says: "Make sure that title is between 50-75 characters." https://www.digitala11y.com/understanding-sc-2-4-2-page-titled/

A lot of our page titles are very lengthy.

**Comment by sarawilcox** [comment]
[Page title - HMRC guidance](https://design.tax.service.gov.uk/hmrc-design-patterns/page-title/)

**Comment by sarawilcox** [comment]
I think we've seen teams struggling with titles, H1s and URLs for fail pages. I'll see if I can find some examples.

**Comment by edwardhorsford** [comment]
I'm a new user of the NHS Design system so recently queried the policy on titles.

There's some guidance under the accessibility section, but alas I didn't find it.

Some extra things it would be good to cover:
* The example seems suited to content pages on nhs.uk, but not services where I think we'd want to include the service name. It would be good to explicitly add an example like the one @sarawilcox has found above.
* As well as services, it would be good to cover what things within the NHS App should do - should they end in `- NHS` or `- NHS App`?

I think the app itself might use `NHS App`, but we may decide that services within it don't - many things are available both within the app and via the web. If we ask them to add `NHS App` to the end of their titles I think it implies the page title being dynamic based on use case - which might be challenging for some services.

**Comment by sarawilcox** [comment]
Interestingly the NHS.UK CMS uses a hyphen by default, but it would not be v difficult to change this to an en-dash.
